tests/test_scheduler.py

```diff
@@ -123,8 +119,8 @@ def test_modify_experiment(self):
         experiment_change = SubmittedExperimentInfo(rid=1, priority=1)
         experiment_new_info = SubmittedExperimentInfo(rid=1, priority=2)
         experiment_delete = SubmittedExperimentInfo(rid=2, priority=1)
-        app.schedulerFrame.model.experimentQueue.append(experiment_delete)
-        app.schedulerFrame.model.experimentQueue.append(experiment_change)
+        app.addExperiment(experiment_delete)
```

Comment:
Please modify the test function name!
I think it should be updated to `test_add_experiment()`.

Reply:
I do not get it, this part tests `app.changeExperiment` too, as a functional test.